src/app/containers/ConsentBanner/index.canonical.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Fragment, useState, useEffect } from 'react';
+import Banner from './Banner';
+import consentBannerUtilities from './CanonicalLogic';
+
+const Canonical = () => {
+  const [showPrivacy, setShowPrivacyBanner] = useState(false);
+  const [showCookie, setShowCookieBanner] = useState(false);
+
+  const {
+    runInitial,
+    privacyOnAllow,
+    privacyOnReject,
+    cookieOnAllow,
+    cookieOnReject,
+  } = consentBannerUtilities({ setShowPrivacyBanner, setShowCookieBanner });
+
+  useEffect(() => {
+    runInitial();
+  }, []);
```

Comment:
I'm sure you've tried, but is it possible to do:

```js
useEffect(runInitial, []);
```

Reply:
I actually thought I tried this in the past but had an error, I stand corrected, Changed :)